package.json

```diff
@@ -11,7 +11,7 @@
   "types": "./dist/lib/index.d.ts",
   "scripts": {
     "start": "node test/echoserver.js",
-    "test": "node test/test.js",
```

Comment:
please keep the oldtest as an `oldtest` script.
I will invoke them after the new tests in CI/CD and if yours pass and the old fails, we see, what needs to be improved. So testing the tests.